linux/scripts/pantos-validator-node-celery

```diff
@@ -55,4 +55,9 @@ if [ "$(id -u)" -ne "$(id -u "$APP_NAME")" ]; then
     EXTRA_ARGS="--uid $(id -u "$APP_NAME")"
 fi
 
-exec ./bin/python -m celery -A pantos.validatornode worker $EXTRA_ARGS -l INFO -n pantos.validatornode -Q pantos.validatornode
+while true; do
+  ./bin/python -m celery -A pantos.validatornode worker $EXTRA_ARGS -l INFO -n pantos.validatornode -Q pantos.validatornode
+
+  echo "Celery worker exited with code $?. Respawning.." >&2
```

Comment:
What's the purpose of this? systemd is already respawning the Celery worker processes.

Reply:
Systemd is indeed restarting celery, but this doesn't happen inside the docker containers as systemd doesn't run there so we need to add this bit.